### PR TITLE
Use staging URL for project generation tests

### PR DESCRIPTION
**Pull request by xorye**
The Travis build now sets a new environment variable: `VSCODE_QUARKUS_API_URL=https://stage.code.quarkus.io/api` for vscode-quarkus to use for the project generation tests.

Signed-off-by: David Kwon <dakwon@redhat.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,6 +17,9 @@ branches:
   only:
     - master
 
+env:
```
Comment (fbricon) [inline .travis.yml]: The env property should be set by the test-ui-run command instead, so it works from CLI everywhere.

**Review by fbricon — CHANGES_REQUESTED** [review]

File: src/test/vscodeUiTest/suite/projectGenerationTest.ts [hunk 2]
```diff
@@ -31,13 +31,19 @@ use(require('chai-fs'));
  */
 describe('Project generation tests', function() {
```
Comment (fbricon) [inline src/test/vscodeUiTest/suite/projectGenerationTest.ts:32]: maybe try process.env.VSCODE_QUARKUS_API_URL=https://stage.code.quarkus.io/api directly here. 
Dunno if it'll work

Comment (xorye) [reply src/test/vscodeUiTest/suite/projectGenerationTest.ts:32]: fixed

File: .travis.yml (hunk 1 above)
Comment (xorye) [reply .travis.yml]: fixed

**Pull request merged**